fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBTypedRecordStore.java

```diff
@@ -169,6 +169,12 @@ public void addRecordWriteConflict(@Nonnull Tuple primaryKey) {
         return untypedStore.scanIndex(index, scanBounds, continuation, scanProperties);
     }
 
+    @Override
+    public RecordCursor<FDBIndexedRecord<Message>> scanIndexPrefetch(Index index, TupleRange range, final KeyExpression commonPrimaryKey,
```

Comment:
`<M>`. Need to pass `serializer` as argument.

Reply:
Done.